docs/api-reference.md

```diff
@@ -17,9 +17,42 @@ Returns the schema of all indexes in JSON.
 curl -XGET localhost:10101/index
 ```
 ``` response
-{"indexes":[{"name":"user","fields":[{"name":"event","options":{"type":"time","timeQuantum":"YMD","keys":false}}]}]}
+{
+    "indexes": [
+        {
+            "fields": [
+                {
+                    "name": "event",
+                    "options": {
+                        "keys": false,
+                        "timeQuantum": "YMD",
+                        "type": "time"
+                    }
+                },
+                {
+                    "name": "language",
+                    "options": {
+                        "cacheSize": 50000,
+                        "cacheType": "ranked",
+                        "keys": false,
+                        "type": "set"
+                    }
+                }
+            ],
+            "name": "user",
+            "options": {
+                "keys": false,
+                "trackExistence": true
+            }
+        }
+    ]
+}
 ```
 
+`GET /schema`
+
+Is equivalent to `GET /index` and returns the same response.
```

Comment:
It seems like we should apply the full description to the `/schema` docs, and put this "equivalent" description under the `/index` docs.